### PR TITLE
Basic Start Time Entry

### PR DESCRIPTION
**Pull request by willsb**
Ref #1143 

This adds basic time entry creation functionality. The target binding for complex queries will come in a different PR.

Squash whenever

### CODE REVIEW

File: Toggl.Giskard/Resources/layout/StartTimeEntryActivity.axml [hunk 1]
```diff
@@ -1,5 +1,98 @@
...
+            android:layout_margin="16dp"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content" />
+    </RelativeLayout>
+    <android.support.design.widget.FloatingActionButton
+        android:id="@+id/StartTimeEntryDoneButton"
+        local:MvxBind="Click DoneCommand"
+        local:elevation="4dp"
+        local:backgroundTint="@color/playButtonGreen"
+        android:src="@drawable/check_light"
```
Comment (kornelijepetak) [inline Toggl.Giskard/Resources/layout/StartTimeEntryActivity.axml]: increasing length order.

File: Toggl.Giskard/Resources/layout/StartTimeEntryActivity.axml [hunk 2]
```diff
@@ -1,5 +1,98 @@
...
+            android:singleLine="true"
+            android:letterSpacing="-0.01"
+            android:lineSpacingExtra="5sp"
+            android:scrollHorizontally="true"
+            android:textColor="@android:color/black"
+            android:background="@android:color/white"
+            android:hint="@string/StartTimeEntryHint"
+            android:textColorHint="@color/defaultText"
+            android:inputType="textNoSuggestions|textVisiblePassword"
+            android:layout_alignParentBottom="true"
```
Comment (kornelijepetak) [inline Toggl.Giskard/Resources/layout/StartTimeEntryActivity.axml]: Increasing length order.

File: Toggl.Giskard/TemplateSelectors/StartTimeEntrySuggestionsTemplateSelector.cs [hunk 3]
```diff
@@ -0,0 +1,64 @@
+﻿using System;
+using MvvmCross.Droid.Support.V7.RecyclerView.ItemTemplates;
+using Toggl.Foundation.Autocomplete.Suggestions;
+using Toggl.Foundation.MvvmCross.Collections;
+
+namespace Toggl.Giskard.TemplateSelectors
+{
+    public sealed class StartTimeEntrySuggestionsTemplateSelector : IMvxTemplateSelector
+    {
+        public const int Empty = 0;
```
Comment (kornelijepetak) [inline Toggl.Giskard/TemplateSelectors/StartTimeEntrySuggestionsTemplateSelector.cs]: Why not enum? I mean, you are basically doing the exact same thing.

Comment (willsb) [reply Toggl.Giskard/TemplateSelectors/StartTimeEntrySuggestionsTemplateSelector.cs]: If that was an enumerable, I'd have to cast it to int, so in the end it'll simply end up being more code.

Comment (kornelijepetak) [reply Toggl.Giskard/TemplateSelectors/StartTimeEntrySuggestionsTemplateSelector.cs]: Maybe I'm not seeing the depth of this, but if enums were used on all places where you are using this, no casting should be necessary. Still, even if casting was needed, a _type_ is a _type_ 😄

**Pull request merged**